queue/src/fiskaltrust.Middleware.Localization.QueueIT/SignProcessorIT.cs

```diff
@@ -3,25 +3,47 @@
 using System.Threading;
 using System.Threading.Tasks;
 using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
 using fiskaltrust.Middleware.Contracts;
+using fiskaltrust.Middleware.Localization.QueueIT.RequestCommands.Factories;
+using fiskaltrust.Middleware.Localization.QueueIT.Services;
 using fiskaltrust.storage.V0;
 using Microsoft.Extensions.Logging;
+using fiskaltrust.Middleware.Contracts.RequestCommands;
+using fiskaltrust.Middleware.Localization.QueueIT.RequestCommands;
+using System.Linq;
 
 namespace fiskaltrust.Middleware.Localization.QueueIT
 {
     public class SignProcessorIT : IMarketSpecificSignProcessor
     {
         private readonly ILogger<SignProcessorIT> _logger;
+        private readonly IRequestCommandFactory _requestCommandFactory;
+        private readonly IITSSCD _client;
+        protected readonly IConfigurationRepository _configurationRepository;
 
-        public SignProcessorIT(            
-            ILogger<SignProcessorIT> logger)
+
+        public SignProcessorIT(
+            ILogger<SignProcessorIT> logger,
+            IRequestCommandFactory requestCommandFactory,
+            IITSSCDProvider itIsscdProvider,
+            IConfigurationRepository configurationRepository)
         {
             _logger = logger;
+            _requestCommandFactory = requestCommandFactory;
+            _client = itIsscdProvider.Instance;
+            _configurationRepository = configurationRepository;
         }
 
-        public Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ProcessAsync(ReceiptRequest request, ftQueue queue, ftQueueItem queueItem)
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ProcessAsync(ReceiptRequest request, ftQueue queue, ftQueueItem queueItem)
         {
-            throw new NotImplementedException();
+            var queueIT = await _configurationRepository.GetQueueITAsync(queue.ftQueueId).ConfigureAwait(false);
+
+            queueIT ??= new ftQueueIT() { ftQueueITId = queue.ftQueueId};
```

Comment:
We should remove this, configuration objects should never be created at runtime. These are configured in the Portal and must be written to the DB at bootstrap time :)